docs/tutorials/data/foph.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "215ad443-34c4-40b1-a6a8-c1f3c667f573",
    "metadata": {},
    "source": [
-    "This notebook provide some examples about how the functions in the `foph` module can be used. "
```

Comment:
Here is a good opportunity to say that these examples are related to COVID datasets only

Reply:
ok